source/2020-05-01-the-ember-times-issue-146.md

```diff
@@ -9,19 +9,31 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
-
+Yehuda's Octane Tutorial continued 🔥,
+  
 READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Yehuda's Octane tutorial continued 🔥](https://yehudakatz.com/2020/03/25/ember-octane-lets-go/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Our very own [Yehuda Katz (@wycats)](https://github.com/wycats) wrote a blog series on [Octane tutorial](https://yehudakatz.com/2020/03/25/ember-octane-lets-go/), and there are two new posts added to it.
```

Comment:
What do you think about the following wording?

```suggestion
Our very own [Yehuda Katz (@wycats)](https://github.com/wycats) wrote a blog series for an [Octane tutorial](https://yehudakatz.com/2020/03/25/ember-octane-lets-go/), and there are two new posts added to it.
```

Reply:
Yeah sounds good now @jayjayjpg .